Include/internal/pycore_code.h

```diff
@@ -24,9 +25,103 @@ struct _PyOpcache {
     char optimized;
 };
 
+
+// We would use an enum if C let us specify the storage type.
+typedef unsigned char _PyFastLocalKind;
+// Note that these all fit within _PyFastLocalKind, as do combinations.
+#define CO_FAST_POSONLY     0x01
+#define CO_FAST_POSORKW     0x02
+#define CO_FAST_VARARGS     0x04
+#define CO_FAST_KWONLY      0x08
+#define CO_FAST_VARKWARGS   0x10
+#define CO_FAST_LOCALONLY   0x20
```

Comment:
Aren't all the above mutually exclusive? Probably best to number the consecutively to make that clear.

Reply:
Aren't comparisons simpler if each has its own bit?  Otherwise `kind & CO_FAST_LOCAL` becomes `KIND == CO_FAST_POSONLY || ... || kind == CO_FAST_LOCAL_ONLY`.

Reply:
Just mask off the cell and free bits, then do a comparison `(kind & VARIABLE_BITS) == CO_FAST_LOCAL_ONLY`

Reply:
The locals-related flags were a holdover from a helper function that I later removed.  I just hadn't removed those flags yet.  I've removed them now, which makes that matter a bit moot.

(FWIW, I expect to add the locals-related flags in a later PR when folding the code object argcounts into `co_localsplus_kinds`.)